backend/open_webui/grounding/wiki_search_utils.py

```
@@ -0,0 +1,574 @@
+            high_quality_results = [r for r in final_results if r["score"] > 0.8]
+
+            return (
+                high_quality_results if high_quality_results else final_results[:3]
+            )  # At least 3 results
+
+        except Exception as e:
+            log.error(f"Search failed: {e}")
+            return []
+
+    async def ground_query(
+        self, query: str, request=None, user=None, messages: List[Dict] = None
```

Ruff (0.12.2)

Unused method argument: `request`

(ARG002)